recipes/gappa/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler:
+  - gcc	# [linux]
+  - clang	# [osx]
+cxx_compiler:
+  - gxx	# [linux]
+  - clangxx	# [osx]
```

Comment:
I assume you tried without this - it's required to build, correct?

Reply:
Yes they are required.